wfoverlap/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, lib, fetchFromGitHub, gfortran, makeWrapper, blas-i8, lapack }:
+assert
+  lib.asserts.assertMsg
+  (blas-i8.isILP64 || blas-i8.passthru.implementation == "mkl")
+  "64 bit integer BLAS implementation required.";
+
+let
+  useMKL64 = blas-i8.passthru.implementation == "mkl";
+  mkl = blas-i8.passthru.provider;
+  blasLibs = if useMKL64
+    then "-lmkl_intel_ilp64 -lmkl_gnu_thread -lmkl_core -lgomp -lpthread -lm -ldl"
+    else "-lblas"
+    ;
+
+in
+  stdenv.mkDerivation rec {
+    pname = "wfoverlap";
+    version = "24.08.2020";
+
+    src =
+      let
+        repo = fetchFromGitHub {
+          owner = "sharc-md";
+          repo = "sharc";
+          rev = "d943ec7aff0fb6c81f61d3c057b0921d053e9e20";
+          sha256 = "1a9frnxvm1jg4cv3jd4lm3q8m7igyc7fsp3baydfjkvk0b4ss9bc";
+        };
+      in "${repo}/wfoverlap/source";
+
+    nativeBuildInputs = [
+      gfortran
+      makeWrapper
+    ];
+
+    buildInputs = [
+      gfortran
+      (if useMKL64 then mkl else blas-i8)
+      lapack
+    ];
+
+    patches = [
+      ./patches/Makefile.patch
+    ];
+
+    postPatch = ''
+       export blasLibs="${blasLibs}"
+       export lapackLibs="-llapack"
+       ls -lah ./Makefile
```

Comment:
This was for debugging purposes?

Reply:
I've used the variable substitute mechanism originally. No longer required.